app/views/work_order_mailer/broker_unconfirmed.html.erb

```diff
@@ -0,0 +1,4 @@
+<p>Hi</p>
+
+<p>RabbitMQ broker is broken on <%= Rails.env %>. </p>
```

Comment:
Should we be more explicit here about why it is broken, i.e. that there are unconfirmed messages?